cmd/vcfFilter/vcfFilter.go

```diff
@@ -16,11 +17,34 @@ import (
 
 func vcfFilter(infile string, outfile string, c criteria, groupFile string, parseFormat bool, parseInfo bool, setSeed int64) (total, removed int) {
 	rand.Seed(setSeed)
-	records, header := vcf.GoReadToChan(infile)
+	var records <-chan vcf.Vcf
+	var header vcf.Header
+	var err error
+	var mapContains bool
+	var sitesSeen map[string]map[int]uint8 = make(map[string]map[int]uint8, 0) //uint8 is the number of times this site is seen in the vcf file.
```

Comment:
Not sure performance is a top priority here, but you might consider making a single map instead of a map of maps. You could wrap `v.Chr` and `v.Pos` in a struct and use that as the map key. 1 lookup is always faster than 2.